components/words.js

```diff
@@ -0,0 +1,8 @@
+export const boardDefault = [
```

Comment:
**fitness for purpose**: why is only this file a `.js` file? Not sure if this was intentional or not.

Reply:
Agreed, all the files should be `.ts` if they do not contain a React component (such as this file) or `.tsx` if they contain a react component.

Reply:
yes it is .ts file now!